README.md

```diff
@@ -223,8 +223,8 @@ You should be able to navigate to `http://localhost:3000` and login with `admin`
 You will need to add a datasource to Grafana. You can do this by navigating to `http://localhost:3000/datasources` and adding a Prometheus datasource. By default, the Prometheus server is running on `http://localhost:9090`. You can use this as the URL for the datasource.
 
 #### Useful Dashboards
-We also provide a set of useful Grafana dashboards which would be useful for monitoring the EigenDA node. You can find them [here](dashboards).
-Once you have Grafana setup, feel free to import the dashboards.
+We also provide a set of useful Grafana dashboards which would be useful for monitoring the EigenDA node. You can find them [here](monitoring/dashboards).
```

Comment:
who is "we" in this context?

Reply:
Replace "We" with name of the specific actor. Is this EigenLabs or EigenDA AVS or something else?

Reply:
Reworded proposal to use a more direct, neutral, and concise tone:

> EigenDA provides a set of Grafana dashboards that provide insights into key performance indicators and health metrics of an EigenDA node. These dashboards can be accessed [here](monitoring/dashboards)

This may need to be reflected in other parts of documentation as well.

Reply:
good catch and rewording. will update.